api/src/pcapi/core/cookies/models.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import asdict
+from dataclasses import dataclass
+from datetime import datetime
+
+import sqlalchemy as sa
+from sqlalchemy import orm
+from sqlalchemy.ext import mutable
+from sqlalchemy.orm import Mapped
+
+from pcapi.models import Base
+from pcapi.models import Model
+from pcapi.models.pc_object import PcObject
+
+
+ListOfCookies = list[str]
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏
```suggestion
CookiesList = list[str] # C'est plus anglais 
```